AnalyticsSampleApp/AppDelegate.swift

```diff
@@ -14,19 +14,31 @@ import UIKit
 import AEPCore
 import AEPIdentity
 import AEPAnalytics
-import AEPAssurance
 import AEPLifecycle
 
+// MARK: TODO remove this once Assurance has tvOS support.
+#if os(iOS)
+import AEPAssurance
+#endif
+
 @UIApplicationMain
+@available(tvOSApplicationExtension, unavailable)
```

Comment:
Is this something that the app developer needs to declare in their App/SceneDelegate when using Analytics ext?

Reply:
Only in the tvOS target.

Reply:
@cdhoffmann do you mind sharing a quick summary of your findings on this and should we add a task to remind us to update the code once this issue is resolved?